### PR TITLE
feat: Don't add `apify_storage` to .gitignore if it already is there

### LINKED ISSUE #72: apify init should only add apify_storage if not already present to .gitignore file

### PR DESCRIPTION
**Pull request by fnesveda**
I had this part of the code open so I decided to just fix the age old issue #72.

Closes #72

### CODE REVIEW

**Comment by metalwarrior665** [comment]
Omg, you are a god!!!

**Review by mnmkng — APPROVED** [review]

File: src/lib/utils.js [hunk 1]
```diff
@@ -237,13 +240,26 @@ const setLocalEnv = async (actDir) => {
...
+    const gitignoreAdditions = [];
+    for (const gitignoreRequirement of GITIGNORE_REQUIRED_CONTENTS) {
+        if (!RegExp(`^${escapeStringRegexp(gitignoreRequirement)}$`, 'mg').test(gitignoreContents)) {
+            gitignoreAdditions.push(gitignoreRequirement);
+        }
+    }
+
+    if (gitignoreAdditions.length > 0) {
+        if (gitignoreContents.length > 0) {
+            fs.writeFileSync(gitignorePath, `\n${gitignoreAdditions.join('\n')}\n`, { flag: 'a' });
```
[the remark this answers is not in the document]
Comment (fnesveda) [reply src/lib/utils.js:259]: Yeah, sure, what do you want it to say?

[the remark this answers is not in the document]
Comment (mnmkng) [reply src/lib/utils.js:259]: Dunno, **# Added by Apify CLI** ?

**Review by drobnikj — APPROVED** [review]
👏

File: src/lib/utils.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (fnesveda) [reply src/lib/utils.js:259]: Okay dokey

**Pull request merged**